### ISSUE TITLE
amp-facebook: Cannot play videos in fullscreen mode

### ISSUE THREAD

**Issue by jmooring** [ampproject/amphtml#28087]
## What's the issue?

The fullscreen button in the video controls does not work.

## How do we reproduce the issue?

Fail:

```html
<amp-facebook
    width="560"
    height="315"
    layout="responsive"
    data-embed-as="video"
    data-href="https://www.facebook.com/nasaearth/videos/10155187938052139/">
</amp-facebook>
```

Fail:

```html
<amp-facebook
    width="560"
    height="315"
    layout="responsive"
    data-allowfullscreen="true"
    data-embed-as="video"
    data-href="https://www.facebook.com/nasaearth/videos/10155187938052139/">
</amp-facebook>
```

Pass (copy + paste of embed code from facebook.com):

```html
<iframe
    src="https://www.facebook.com/plugins/video.php?href=https%3A%2F%2Fwww.facebook.com%2Fnasaearth%2Fvideos%2F10155187938052139%2F&show_text=0&width=560" 
    width="560"
    height="315"
    style="border:none;overflow:hidden"
    scrolling="no"
    frameborder="0"
    allowTransparency="true"
    allowFullScreen="true">
</iframe>
```

## What browsers are affected?

* Google Chrome Version 81.0.4044.122 (Official Build) (64-bit)
* Perhaps all

## Which AMP version is affected?

`Powered by AMP ⚡ HTML – Version 2004172112280`
